README.md

```diff
@@ -85,6 +85,12 @@ If you decide to build Emacs from source there's some useful information on this
 - [Emacs tree-sitter starter-guide](https://git.savannah.gnu.org/cgit/emacs.git/tree/admin/notes/tree-sitter/starter-guide?h=emacs-29)
 - [Emacs install instructions](https://git.savannah.gnu.org/cgit/emacs.git/tree/INSTALL.REPO).
 
+To check if your emacs already supports tree sitter run
+
+``` emacs-lisp
+(treesit-available-p)
```

Comment:
I'm guessing we should also check this when loading the mode, if we're not doing it already. :-)